.releaserc.yml

```diff
@@ -9,4 +9,4 @@ plugins:
 
   # use the publish hook of semantic-release to publish the modules to the nexus
   - - "@semantic-release/exec"
-    - publishCmd: "export SEMANTIC_VERSION=${nextRelease.version} && ./gradlew publish"
+    - publishCmd: "export SEMANTIC_VERSION=${nextRelease.version} && ./gradlew publishMavenPublicationToMavenCentralRepository closeAndReleaseRepository publishMavenPublicationToSdaInternRepository"
```

Comment:
Do we have some kind of dry run? Or a manual test without affecting our master branch?

Reply:
https://search.maven.org/search?q=org.sdase.commons

Reply:
Ok, let's give it a try.